__tests__/warnings.spec.ts

```diff
@@ -27,6 +27,7 @@ describe('warnings', () => {
     })
     router.resolve({ path: '/', params: { p: 'p' } })
     expect('Path "/" was passed with params').toHaveBeenWarned()
+    expect(`No route matched`).toHaveBeenWarned()
```

Comment:
let's avoid the other warning instead

```suggestion
    router.resolve({ path: '/p', params: { p: 'p' } })
    expect('Path "/p" was passed with params').toHaveBeenWarned()
```